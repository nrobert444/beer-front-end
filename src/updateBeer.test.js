import React from 'react';
import UpdateBeer from './updateBeer.js';
import renderer from 'react-test-renderer';

it('UpdateBeer renders correctly', () => {
  const tree = renderer
    .create(<UpdateBeer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});