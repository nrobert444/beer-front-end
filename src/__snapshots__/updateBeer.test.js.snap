// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateBeer renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    Update a beer!
    <br />
    <label>
      Name:
      <input
        onChange={[Function]}
      />
    </label>
    <br />
    <label>
      Style:
      <select
        onChange={[Function]}
      />
    </label>
    <br />
    <label>
      Brewery:
      <input
        onChange={[Function]}
      />
    </label>
    <br />
    <label>
      ABV:
      <input
        onChange={[Function]}
        type="number"
      />
    </label>
    <br />
    <label>
      Image:
      <input
        onChange={[Function]}
      />
    </label>
    <br />
    <label>
      Is in season:
      <select
        onChange={[Function]}
      >
        <option
          value="true"
        >
          True
        </option>
        <option
          value="false"
        >
          False
        </option>
      </select>
    </label>
    <br />
    <button>
      Submit
    </button>
  </form>
  <button
    onClick={[Function]}
  >
    Delete
  </button>
</div>
`;
